coalib/settings/SectionFilling.py

```diff
@@ -20,14 +21,20 @@ def fill_settings(sections, acquire_settings, log_printer):
                              description in [0] and the names of the bears
                              who need this setting in all following indexes.
     :param log_printer:      The log printer to use for logging.
-    :return:                 A tuple containing (local_bears, global_bears),
-                             each of them being a dictionary with the section
-                             name as key and as value the bears as a list.
+    :return:                 A tuple containing (local_bears, global_bears,
+                             aspects), each of them being a dictionary with the
+                             section name as key and as value the bears as a
+                             list.
     """
     local_bears = {}
     global_bears = {}
+    aspects = {}
 
     for section_name, section in sections.items():
+        section_aspects = extract_aspects_from_section(section)
+        if section_aspects is not None:
+            # Not required anymore and avoiding name conflict.
```

Comment:
It is to avoid conflict when running bear. `Bear::run_bear_from_section` will override `aspects` from here with what it find from `section['aspects']`.